### PR TITLE
Migrations (Part 2)

### PR DESCRIPTION
**Pull request by slifty**
This is the second of several PRs related to the table migrations from issue #254

### CODE REVIEW

**Comment by cecilia-donnelly** [comment]
Hey @slifty I'm investigating these test errors and it looks like the quotes in the csv's are being imported as part of the data.  I removed them and got further in the process, but their absence may (?) cause other issues later -- I'm getting "Entity type specified is not recognized" for various values of entity structure on an organizational enrollment.  I'll report back once I've tracked that down.

**Comment by slifty** [comment]
@cecilia-donnelly A little surprised to hear about the quotes -- I thought quoted fields was part of the CSV spec.

Ideally we would keep the quotes, but figure out why they are not being properly parsed as CSV.

**Comment by cecilia-donnelly** [comment]
@slifty it looks like [this person had the same problem](http://forum.liquibase.org/topic/use-of-quotation-marks-when-oading-data-from-csv-file), but there is unfortunately no resolution there.  Also, it was 7 years ago, so I'd hope any problem would have been fixed by now!

All of which is to say that I haven't figured this out yet. >_<

**Comment by slifty** [comment]
In case we face this problem again, the issue was that there was whitespace between the comma and the opening quotes (e.g. `"a", "b"` instead of `"a","b"`)

Looks like things are fixed now; I'll merge when I get a 👍 from someone!

File: psm-app/db/seed.sql [hunk 1]
```diff
@@ -38,34 +38,6 @@ DROP TABLE IF EXISTS
...
-  ('17', 'Consumer Directed Community Support (CDCS)'),
-  ('18', 'Specialized Supplies & Equipment (Waiver)'),
-  ('19', 'Spenddown Collection'),
-  ('20', 'Customized Living, 24 CL, Residential Care Services, Assisted Living Services'),
-  ('21', 'Structured Day Program Services'),
-  ('22', 'Supported Employment Services'),
-  ('23', 'Family Counseling and Training'),
-  ('24', 'Supported Living Services'),
-  ('25', 'Foster Care Services'),
-  ('26', 'Waiver Transportation');
```
Comment (cecilia-donnelly) [inline psm-app/db/seed.sql]: So `service_categories` and `categories_of_service` are really exactly the same?

Comment (cecilia-donnelly) [reply psm-app/db/seed.sql]: (Not a criticism, to be clear, just marveling.)

Comment (slifty) [reply psm-app/db/seed.sql]: They are -- and it drove me nuts to have to keep it but it looks like both are used in different places :(

**Pull request merged**